cupy_backends/cuda/api/runtime.pyx

```diff
@@ -1114,6 +1114,12 @@ cpdef graphUpload(intptr_t graphExec, intptr_t stream):
         status = cudaGraphUpload(<GraphExec>(graphExec), <driver.Stream>stream)
     check_status(status)
 
+cpdef graphDebugDotPrint(intptr_t graph, str path, unsigned int flags):
+    path_byte = path.encode('ascii')
```

Comment:
I've tested this one and found this fails when containing non-ASCII character, e.g. `export TMP=Testå`

```suggestion
    path_byte = path.encode()
```